tests/testthat/test-utils-read.R

```diff
@@ -201,7 +201,7 @@ test_that('readApprovalPoints data returns as expected', {
 
 test_that('readApprovalBar data returns as expected', {
   library("jsonlite")
-  ts <- fromJSON('{
+  ts <- repgen:::readTimeSeries(fromJSON('{ "anExampleSeries" : {
```

Comment:
I think you want to skip this and just manually create the things needed in `ts` for `readApprovalBar`? Because we aren't also trying to test `readTimeSeries` here